templates/searchadmin/search/quicksearch/info_table.html.twig

```diff
@@ -49,15 +49,17 @@
             {% endif %}
         {% endif %}
 
-        {% set owner = pimcore_user(element.getUserOwner()) %}
+        {% set userOwnerId = element.getUserOwner() %}
+        {% set owner = userOwnerId ? pimcore_user(userOwnerId) : null %}
         {% if owner is instanceof('\\Pimcore\\Model\\User') %}
             <tr>
                 <th>{{ 'owner'|trans([],'admin') }}</th>
```

Comment:
this looks like it would break the table column on the rows where the userOwner is missing